testbed_example/example_optical_model.py

```diff
@@ -30,6 +30,10 @@ def pupil_grid(self):
 
         return hcipy.make_uniform_grid(dims, size)
 
+    @property
+    def focal_grid(self):
+        return hcipy.make_focal_grid(12, 32)
+
```

Comment:
Do you think wherever you needed the `focal_grid` you could just use the `detector_grid` below? I would like the grid parameters to be read from the configuration, and I was trying to call things in the detector plane (where the focal grid is) with "detector" to keep things more or less consistent.